opsdroid/configuration/schema.yaml

```diff
@@ -0,0 +1,118 @@
+welcome-message: bool(required=False)
+
+connectors:
+  list(include('websocket', required=False),
+  include('shell', required=False),
+  include('ciscospark',required=False),
+  include('facebook',required=False),
+  include('github',required=False),
+  include('matrix',required=False),
+  include('slack',required=False),
+  include('telegram',required=False),
+  include('twitter',required=False))
+
+databases:
+  list(include('mongo', required=False),
+  include('redis', required=False),
+  include('sqlite', required=False),required=False)
+
+skills: list(include('skill',required=False))
+
+parsers: list(include('parser',required=False),required=False)
+---
+mongo:
+  name: regex('^mongo$')
```

Comment:
Just a question, you are setting that mongo database should be written in all lower case right? 
I'm inclined to say that we should allow users to have the first letter in uppercase, mostly because sometimes I forget and I write `Mongo` or `mongo`. 

What do you think about this?

Reply:
Now it is possible to write case-insensitive names!  To test this test is added: "test_load_config_valid_case_sensitivity"

Reply:
Thats awesome thanks a lot for working on this 👍